src/components/Lesson/Lesson.js

```diff
@@ -103,6 +103,8 @@ const Lesson = React.createClass({
     //this.setLanguage();
   },
   componentDidMount() {
```

Comment:
Men nå ser jeg, etter å ha testet med `yarn build`, at det ikke funker for statiske sider.

Etter å ha googlet litt, så foreslår jeg at du bruker https://github.com/gaearon/react-document-title

Hvis du synes det blir for komplisert, så gi beskjed, så skal jeg lage PR'en.